README.md

```diff
@@ -18,7 +18,29 @@
 
 ## Background
 
-TBD
+This is useful for unit tests, in Node.js and the browser.
```

Comment:
If this is for unit testing, can its name reflect that instead?

Reply:
It's not /just/ for unit tests tho, we have a lot of libraries that pre-load these contexts. It's basically to serve the same place in the ecosystem as bedrock-security-context, but isomorphic.

Reply:
A "core-document-loader" sounds like something you should always load. But what you should always load depends on your application. Just because we happen to have a number of applications that load a subset of contexts does not mean that those should be considered core *for all applications*. So if this isn't just for testing, what's the better name for this? What is this for -- security-based applications? security-document-loader?

Reply:
How about "common digitalbazaar project documentLoader"?

Reply:
> How about "common digitalbazaar project documentLoader"?

That doesn't address the "core" issue :P ... it doesn't say what it is. This is a special bundled doc loader where what's in the *bundle* is security-related contexts. So the name should reflect that somehow.

Reply:
@dlongley sure. so I'm proposing to remove 'core' and name it something like 'common-security-document-loader'

Reply:
I'm ok with `common-security-document-loader` ... but `security-document-loader` *is* shorter.

Reply:
That's true! will change it to `security-document-loader`!

Reply:
@dlongley renamed.